common/autoware_test_utils/include/autoware_test_utils/autoware_test_utils.hpp

```diff
@@ -16,11 +16,11 @@
 #define AUTOWARE_TEST_UTILS__AUTOWARE_TEST_UTILS_HPP_
 
 #include <autoware/universe_utils/geometry/geometry.hpp>
+#include <autoware_lanelet2_extension/io/autoware_osm_parser.hpp>
```

Comment:
[The documentation](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/) suggests creating a directory structure like `include/autoware/***`. If you are considering adding a prefix, I think it might be a good opportunity to revise the structure as well.

Reply:
I see. I will do it. Please hold on reviewing.

Reply:
Since this PR involves a breaking change due to a major version update, I still want to limit it to just renaming. Is it okay to change the include file structure in the next PR?

Reply:
Okay, let's leave the directory structure as is for now :+1: